lib/saml_idp_constants.rb

```diff
@@ -5,11 +5,6 @@ module Constants
       LOA1_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/1'.freeze
       LOA3_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/3'.freeze
       REQUESTED_ATTRIBUTES_CLASSREF = 'http://idmanagement.gov/ns/requested_attributes?ReqAttr='.freeze
-
```

Comment:
My assumption was that constants were constant (i.e. parsed once at compile time, not per-request). Doesn't matter what file they are in, right? Maybe my understanding is flawed.